account-kit/infra/src/client/smartAccountClient.ts

```diff
@@ -174,21 +171,26 @@ export function createAlchemySmartAccountClient(
       ? alchemyUserOperationSimulator(config.transport)
       : undefined,
     signUserOperation: config.signUserOperation,
+    // @ts-expect-error
+    addBreadcrumb(crumb: string) {
+      const oldConfig = config.transport.config;
+      const dynamicFetchOptions = config.transport.dynamicFetchOptions;
```

Comment:
dynamic fetch options is the mutable headers, but these are created per new instance of alchemyTransport